src/main/java/uk/gov/hmcts/cmc/claimstore/ClaimStoreApplication.java

```diff
@@ -3,11 +3,14 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
 import org.springframework.cloud.netflix.feign.EnableFeignClients;
 
-@SpringBootApplication(exclude = DataSourceAutoConfiguration.class)
+@SpringBootApplication(exclude = DataSourceAutoConfiguration.class,
+    scanBasePackages = {"uk.gov.hmcts.cmc.claimstore", "uk.gov.hmcts.document"})
```

Comment:
why is this scanning document?

Reply:
removed

Reply:
https://github.com/hmcts/http-proxy-spring-boot-autoconfigure/blob/master/src/main/resources/META-INF/spring.factories#L1